app/models/journeyDomain/routing/CountryOfRoutingDomain.scala

```diff
@@ -34,7 +35,11 @@ case class CountryOfRoutingDomain(
       case AccessingJourney =>
         Some(controllers.routing.index.routes.CountryOfRoutingController.onPageLoad(userAnswers.lrn, mode, index))
       case CompletingJourney =>
-        CountriesOfRoutingSection.route(userAnswers, mode)
+        mode match {
+          case NormalMode => CountriesOfRoutingSection.route(userAnswers, mode)
+          case CheckMode  => RouteDetailsSection.route(userAnswers, mode)
+        }
```

Comment:
I think this can be reverted along with the `CountryOfRoutingNavigator` change, but would be good to understand why this change was made?